src/AzureOperator.ts

```diff
@@ -56,6 +57,12 @@ export class AzureOperator {
   async Deploy(
       context: vscode.ExtensionContext, channel: vscode.OutputChannel,
       telemetryContext: TelemetryContext) {
+    if (RemoteExtension.isRemote(context)) {
+      const message =
+          `You are in a container now. 'Azure IoT Device Workbench: Depoly to Azure...' command is currently not supported inside the container.`;

```

Comment:
The project is currently open in container now. 'Azure IoT Device Workbench: Depoly to Azure...' is not supported inside the container.

Reply:
Fixed.